package org.serratec.excecoes;

public class CpfRepetidoException extends Exception {
	public CpfRepetidoException() {
		super("CPF já cadastrado no sistema.");
	}
}
